typescript/packages/subsurface-viewer/src/layers/grid3d/fragment.fs.glsl.ts

```diff
@@ -37,11 +39,11 @@ vec4 getPropertyColor (float propertyValue) {
          // Use min/max color to clamp.
          x = max(0.0, x);
          x = min(1.0, x);
-         color = texture2D(colormap, vec2(x, 0.5));
+         color = texture2D(colormap, vec2(x + colorLookupTolerance , 0.5));
```

Comment:
Is this really needed ? x is clamped to [0, 1]

Reply:
The shader has been reworked for discrete coloring